Failed tests (12468): TestChaos

The following test appears to have failed:

[#12468](https://circleci.com/gh/cockroachdb/cockroach/12468):

```
E0208 02:03:54.259263 27426 acceptance/cluster/localcluster.go:410  node=0 status=restart
I0208 02:03:54.262949 27426 acceptance/chaos_test.go:162  client 0 shutting down
I0208 02:03:54.426406 27426 acceptance/chaos_test.go:162  client 1 shutting down
I0208 02:03:54.448069 27426 acceptance/chaos_test.go:162  client 2 shutting down
I0208 02:03:54.550401 27426 acceptance/cluster/localcluster.go:538  stopping
--- FAIL: TestChaos (18.39s)
	chaos_test.go:280: dial tcp 0.0.0.0:32787: getsockopt: connection refused
=== RUN   TestGossipPeerings
I0208 02:03:57.242309 27426 acceptance/cluster/localcluster.go:219  started docker-spy: 172.17.0.16
I0208 02:03:57.242338 27426 acceptance/cluster/localcluster.go:226  initializing cluster
I0208 02:03:59.100442 27426 acceptance/cluster/localcluster.go:333  creating ca (1024bit) in: /home/ubuntu/cockroach/.localcluster.certs.822025371
I0208 02:03:59.158545 27426 acceptance/cluster/localcluster.go:338  creating node (1024bit) certs in: /home/ubuntu/cockroach/.localcluster.certs.822025371
I0208 02:04:01.466503 27426 acceptance/cluster/localcluster.go:389  *** started roach0.local ***
  ui:    https://0.0.0.0:32789
  trace: https://0.0.0.0:32789/debug/requests
  logs:  /tmp/circle-artifacts.c2nnytD/acceptance/TestGossipPeerings/roach0.local/cockroach.INFO
--
--- SKIP: TestBuildBabyCluster (0.00s)
	terraform_test.go:32: only enabled during testing
=== RUN   TestFiveNodesAndWriters
--- SKIP: TestFiveNodesAndWriters (0.00s)
	main_test.go:62: running in docker mode
FAIL
ok github.com/cockroachdb/cockroach/acceptance 1337s

```
Please assign, take a look and update the issue accordingly.

--- a/acceptance/chaos_test.go
+++ b/acceptance/chaos_test.go
@@ -272,13 +272,14 @@ CREATE TABLE bank.accounts (
 
 	// Verify accounts.
 
-	// Recreate db client because node 0 might have been restarted by
+	// Hold the read lock on node 0 to prevent it being restarted by
 	// chaos monkey.
-	db = makePGClient(t, c.PGUrl(0))
 	var sum int
-	if err := db.QueryRow("SELECT SUM(balance) FROM bank.accounts").Scan(&sum); err != nil {
+	clients[0].RLock()
+	if err := clients[0].db.QueryRow("SELECT SUM(balance) FROM bank.accounts").Scan(&sum); err != nil {
 		t.Fatal(err)
 	}
+	clients[0].RUnlock()
 	if sum != 0 {
 		t.Fatalf("The bank is not in good order. Total value: %d", sum)
 	}

